client/App.jsx

```diff
@@ -27,7 +27,7 @@ class App extends React.Component {
     this.initialize();
   }
 
-  initialize(productId = '22122') {
+  initialize(productId = '22126') {
```

Comment:
I don't mind switching to a new product_id (we can also double check with everyone else too), but I'm not sure if this was intentionally left there. Just want to point out! :)

Reply:
I switched it because I needed  a product with as many images as possible. I can switch it back now that I know it works. My tests just won't pass unless this productId is used